app/src/test/java/com/kickstarter/viewmodels/CreatorDashboardViewModelTest.java

```diff
@@ -58,9 +58,11 @@ public void testStartProjectActivity() {
   }
 
   @Test
-  public void testLatestProjectAndStats() {
+  public void testProjectAndStats() {
+    final Project projectOne = ProjectFactory.project();
+    final Project projectTwo = ProjectFactory.project();
```

Comment:
maybe do a builder with these and just change something like the `id()` to `id(1)` and `id(2)` or something to make the validity of this test stronger (since `projectOne` and `projectTwo`) are technically identical 👯

Reply:
👌 I think the `public static synchronized int` magic in the `IdFactory` does this for us but also fine with being more explicit.

Reply:
ahhh yeah you're right! forgot about that nice lil helper, fine with either way, ty ty